dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.managedBeansWar-1.0.feature

```diff
@@ -3,8 +3,8 @@ symbolicName=com.ibm.websphere.appserver.managedBeansWar-1.0
 IBM-App-ForceRestart: install, \
  uninstall
 IBM-Provision-Capability: \
-  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.managedBeansCore-1.0))", \
-  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=com.ibm.websphere.appserver.servlet-3.0)(osgi.identity=com.ibm.websphere.appserver.servlet-3.1)(osgi.identity=com.ibm.websphere.appserver.servlet-4.0)))"
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=com.ibm.websphere.appserver.managedBeansCore-1.0)(osgi.identity=io.openliberty.managedBeansCore-2.0)))", \
```

Comment:
I think that this filter is wrong.  I think it should be
(1.0 & (3.0 | 3.1 | 4.0)) | (2.0 & 5.0)

the way that the filter is currently written it would allow 1.0 and 5.0 to be honored with the filter and that is not right.  The other feature checking will prevent that, but it would be good to get it written correctly.  As it is I am ok without this PR being done as well and just to leave the auto feature.

In review of another PR, I suggested leaving them alone even though they enable the same bundle.  For that PR the two auto features were:  com.ibm.websphere.appserver.jaxwscdi-2.2 and com.ibm.websphere.appserver.jaxwscdi-2.3

Reply:
Based off of discussions outside of the PR about this concern, I am going to mark this resolved and approve the PR.